src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py

```diff
@@ -96,7 +96,7 @@ def _translate_schedule(
             }
         return {
             "schedule_interval": "@once",
-           # set the a start time in the past and disable catchup so airflow runs the dag immediately
+            # set the a start time in the past and disable catchup so airflow runs the dag immediately
```

Comment:
```suggestion
           # set the start time in the past and disable catchup so airflow runs the dag immediately
```

Wasn't sure of the purpose of this additional space here...

Reply:
the format script automatically formatted it like this, seems like the line was wrongly indented before.

Reply:
just checked the file, indentation looks correct, so will mark this resolved